Packages/com.unity.inputsystem/InputSystem/Events/IMECompositionEvent.cs

```diff
@@ -34,7 +34,7 @@ public FourCC GetTypeStatic()
 namespace UnityEngine.Experimental.Input
 {
     [StructLayout(LayoutKind.Explicit, Size = sizeof(int) + sizeof(char) * IMECompositionEvent.kIMECharBufferSize)]
-    public unsafe struct IMECompositionString : IEnumerable<char>
+    public unsafe struct IMEComposition : IEnumerable<char>
```

Comment:
Is the revert intentional?

Reply:
Yes, since it's not a string under the hood or in how it's accessed, I removed all references to the word string, except in ToString.

Reply:
Heh in that case, please revert :) It's an IEnumerable<char> which makes it a string.